src/openfermion/hamiltonians/_molecular_data.py

```diff
@@ -36,7 +35,7 @@
 # Define a compatible basestring for checking between Python 2 and 3
 try:
```

Comment:
What is this doing? I'm not criticizing it, just asking a question.

Reply:
It's a compatibility fix for data types in hdf5.  Python2 and 3 have different default string classes and it causes problems between the versions. The no cover statement is because I don't see how to get both clauses to execute in both Python2 and 3, which means in both it lowers the test coverage by 1 line, however both lines are run in the tests separately, so it should be okay.